src/test/java/io/arlas/server/DataSetTool.java

```diff
@@ -51,6 +57,19 @@
     public final static String DATASET_TIMESTAMP_FORMAT="epoch_millis";
     public static final String[] jobs= {"Actor", "Announcers", "Archeologists", "Architect", "Brain Scientist", "Chemist", "Coach", "Coder", "Cost Estimator", "Dancer", "Drafter"};
 
+    public  static  Object jsonSchema;
+
+    static {
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            jsonSchema = mapper.readValue(classLoader.getResourceAsStream("dataset.schema.json"),ObjectNode.class);
+            System.out.println("STATIC");
+            System.out.println(jsonSchema);
+
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
You should use logger instead of printStackTrace().

Reply:
Done